cypress/integration/article-view-spec.js

```diff
@@ -56,10 +56,10 @@ describe('Article view', () => {
   it('check quick facts opens', () => {
     goToCatArticle()
     articlePage.selectOptionFromActionsMenu('quickfacts')
-    quickFactsPage.table().should('contains.text', 'Various types of domestic cat')
+    quickFactsPage.table().should('contains.text', 'Various types of the domestic cat')
```

Comment:
@jpita FYI I updated this test spec as someone changed the article content over the weekend and added a "the" to the quick facts:

https://en.wikipedia.org/w/index.php?title=Cat&action=history
https://en.wikipedia.org/w/index.php?title=Cat&oldid=944595654

Reply:
I'm putting this commit on master, otherwise keeps getting test spec error when developing others.

Reply:
Sounds good. I'm assuming you are talking about [a `git cherry-pick`](https://mattstauffer.com/blog/how-to-merge-only-specific-commits-from-a-pull-request/) or something like that?

Reply:
done and yes cherry-pick